plugins/chat/assets/javascripts/discourse/components/channels-list-direct.gjs

```diff
@@ -60,8 +88,7 @@ export default class ChannelsListDirect extends Component {
     {{#if
       (and
         this.showDirectMessageChannels
-        this.site.desktopView
-        this.chatStateManager.isDrawerActive
+        (or this.site.desktopView (not this.publicChannelsDisabled))
```

Comment:
Maybe we can prevent a double negative here by updating to `publicChannelsEnabled` instead to make this easier to read.